backend/src/openarchiefbeheer/destruction/tests/test_models.py

```diff
@@ -686,3 +688,36 @@ def test_clean_local_metadata(self):
 
         self.assertEqual(item1.extra_zaak_data, {})
         self.assertEqual(item2.extra_zaak_data, {})
+
+
+class DestructionListCoReviewTest(TestCase):
+    def test_create(self):
+        DestructionListCoReviewFactory.create()
+
+    def test_created(self):
+        co_review = DestructionListCoReviewFactory.create()
+        self.assertTrue(co_review.created)
+
+    def test_destruction_list_hierarchy(self):
+        co_review = DestructionListCoReviewFactory.create()
+        destruction_list = co_review.destruction_list
+        self.assertIn(co_review, destruction_list.co_reviews.all())
+
+    def test_author_list_hierarchy(self):
+        co_review = DestructionListCoReviewFactory.create()
+        author = co_review.author
+        self.assertIn(co_review, author.created_co_reviews.all())
+
+    def test_str(self):
+        destruction_list = DestructionListFactory(name="Destruction list to co-review")
+        author = UserFactory.create(
+            username="co-reviewer", first_name="Co", last_name="Reviewer"
+        )
+        co_review = DestructionListCoReviewFactory.create(
+            destruction_list=destruction_list, author=author
+        )
+
+        self.assertEqual(
+            str(co_review),
+            "Co-review for Destruction list to co-review (Co Reviewer (co-reviewer))",
+        )
```

Comment:
Since this is a test for the factory and not for the model, I would move it to test_test_utilities.py (...or some better name :stuck_out_tongue: )

Reply:
It's not necessarily a test for the factory (at least not intended :P). I tried to test primarily the auto fields and whether all relations work properly.

Reply:
Ah I see :sweat_smile: